### PR TITLE
pythonPackages.papermill: add missing dependencies

### PR DESCRIPTION
**Pull request by lsix**
###### Motivation for this change

I noticed the build was failing

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/nbclient/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  pname = "nbclient";
+  version = "0.4.0";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1id6m2dllkjpbv2w0yazxhlkhdd9cac6lxv9csf053il9wq322lk";
+  };
+
+  buildInputs = [ pytest xmltodict nbconvert ipywidgets ];
```
Comment (jonringer) [inline pkgs/development/python-modules/nbclient/default.nix:16]:
```suggestion
  checkInputs = [ pytest xmltodict nbconvert ipywidgets ];
```

**Review by jonringer — COMMENTED** [review]
otherwise LGTM

File: pkgs/development/python-modules/nbclient/default.nix (hunk 1 above)
Comment (lsix) [reply pkgs/development/python-modules/nbclient/default.nix]: Fixed. Thanks for spotting it.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 91481 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>4 packages built:</summary><br>- python37Packages.nbclient<br>- python37Packages.papermill<br>- python38Packages.nbclient<br>- python38Packages.papermill</details>

**Pull request merged**